dnsimple/service/registrar.py

```diff
@@ -418,3 +418,45 @@ def renew_whois_privacy(self, account_id, domain):
         """
         response = self.client.post(f'/{account_id}/registrar/domains/{domain}/whois_privacy')
         return Response(response, WhoisPrivacyRenewal)
+
+    def get_domain_transfer_lock(self, account: int, domain: str):
```

Comment:
Adding type hints will break any integrations that are using Python < 3.5. While we do not advertise support for versions prior to 3.8, we do say prior versions may work. It may be worth adding a note in the changelog for explicitness.

Reply:
Where do we say that prior versions may work? I can only see on the README that we mention later versions may work, but not prior versions, and I can't find anywhere else that mentions support.

Reply:
I've decided to remove the type hints for now, since it's not consistent with the existing code.